sites/all/themes/humanitarianresponse/assets/sass/partials/_panel.scss

```diff
@@ -109,6 +109,23 @@
   }
 }
 
+/* hid contact name font style */
+
+.field-items {
```

Comment:
Every single field will have this class, not only the hid contact reference one.